package-system/AWSNativeSDK/build_AWSNativeSDK_android.cmd

```diff
@@ -0,0 +1,81 @@
+@echo off
+REM 
+REM Copyright (c) Contributors to the Open 3D Engine Project.
+REM For complete copyright and license terms please see the LICENSE at the root of this distribution.
+REM 
+REM SPDX-License-Identifier: Apache-2.0 OR MIT
+REM 
+
+SET SRC_PATH=temp\src
+SET BLD_PATH=temp\build
+
+IF "%ANDROID_NDK_ROOT%"=="" (
+    ECHO "Required envrironment variable ANDROID_NDK_ROOT is missing, please set it to local android ndk directory"
+    exit /b 1
+)
+
+REM Debug Shared
+call:ConfigureAndBuild Debug Shared
+IF %ERRORLEVEL% NEQ 0 (
+    exit /b 1
+)
+
+REM Debug Static
+call:ConfigureAndBuild Debug Static
+IF %ERRORLEVEL% NEQ 0 (
+    exit /b 1
+)
+
+REM Release Shared
+call:ConfigureAndBuild Release Shared
+IF %ERRORLEVEL% NEQ 0 (
+    exit /b 1
+)
+
+REM Release Static
+call:ConfigureAndBuild Release Static
+IF %ERRORLEVEL% NEQ 0 (
+    exit /b 1
+)
+
+ECHO "Custom Build for AWSNativeSDK finished successfully"
+exit /b 0
+
+:ConfigureAndBuild
+SET BUILD_TYPE=%~1
+SET LIB_TYPE=%~2
+SET BUILD_SHARED=OFF
+IF %LIB_TYPE% EQU Shared (
+    SET BUILD_SHARED=ON
+)
+ECHO "CMake Configure %BUILD_TYPE% %LIB_TYPE%"
+call cmake -S %SRC_PATH% -B %BLD_PATH%\%BUILD_TYPE%_%LIB_TYPE% ^
+           -G Ninja ^
+           -DNDK_DIR="%ANDROID_NDK_ROOT%" ^
+           -DBUILD_SHARED_LIBS=%BUILD_SHARED% ^
+           -DCMAKE_BUILD_TYPE="%BUILD_TYPE%" ^
+           -DTARGET_ARCH=ANDROID ^
+           -DANDROID_NATIVE_API_LEVEL=21 ^
```

Comment:
I wonder if theres some overall value to track here, rather than each 3P defining its only minimal Android version?

Reply:
not sure, but it is minimal level required by aws sdk cpp https://github.com/aws/aws-sdk-cpp/blob/main/Docs/CMake_Parameters.md#android_native_api_level